Stack/Opc.Ua.Bindings.Https/Stack/Https/HttpsTransportListener.cs

```diff
@@ -234,7 +234,7 @@ public void Start()
             httpsOptions.CheckCertificateRevocation = false;
             httpsOptions.ClientCertificateMode = ClientCertificateMode.NoCertificate;
             httpsOptions.ServerCertificate = m_serverCert;
-            httpsOptions.SslProtocols = SslProtocols.Tls | SslProtocols.Tls11 | SslProtocols.Tls12;
+            httpsOptions.SslProtocols = SslProtocols.None;
```

Comment:
This is a functional change from Tlx|Tls11|Tls12 to None, what is the rational behind?

Reply:
I think that setting "_SslProtocols = SslProtocols.None_" will allow the OS to chose the best protocol to use.
But maybe Martin can confirm...

Reply:
exactly, Tls 1.1 etc should not be used anymore